packages/client/src/recentlyViewed/types/deleteRecentlyViewedProducts.types.ts

```diff
@@ -0,0 +1,6 @@
+import type { Config } from '../../types';
+
+export type DeleteRecentlyViewedProducts = (
+  id: number,
+  config?: Config,
+) => Promise<unknown>;
```

Comment:
Question: Is this unknown? At least, it should return the request status, I believe.

Reply:
This should be void, because as I check we don't get anything from response, only request status matters.

Reply:
Actually disregard this, you are right, it should be Promise< void > because the .then is returning the data field only and not all the response object.